backend/dao/usersDAO.js

```diff
@@ -63,18 +55,37 @@ module.exports = class UsersDAO {
   }
   static async addUser(user_id, username, name, email) {
     //review takes all the parameters ^^ and creates a user document
+    //todo: add list type to json object value... list for the wines the user saves to their profile.
     try {
       const userDoc = {
         user_id: user_id,
         username: username,
         name: name,
         email: email,
       };
-
-      //console.log(userDoc);
       return await users.insertOne(userDoc);
     } catch (e) {
       console.error(`Unable to post user: ${e}`);
     }
   }
+  static async updateUser(user_id, username, id) {

```

Comment:
Why do we have the `id` parameter?
I don't see it being used.
The linter should catch this and throw errors, do we have `eslint` for the BE code?
If not I can explain what is it for.

Reply:
oh I think I was testing this out with the IDs, and id param is not used... but stopped working and editing on the user routes since these are no longer in our MVP, and moved onto the wines portion instead. I paused any work or updating, code cleaning, that is in users, until the wines are finished to fit the MVP

Reply:
But to confirm, do we have `eslint` for the project?

Reply:
ah no I do not have this..